### PR TITLE
Upgrade webpack to v5

### PR DESCRIPTION
**Pull request by acrellin**
Upgrade webpack to v5.x and browserify non-polyfilled node modules for
v5 compatibility.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -85,6 +86,9 @@ const config = {
   plugins: [
     // Uncomment the following line to enable bundle size analysis
     //    new BundleAnalyzerPlugin()
+    new webpack.ProvidePlugin({
```
Comment (stefanv) [inline webpack.config.js:89]: What is this plugin for?

File: webpack.config.js [hunk 2]
```diff
@@ -103,7 +107,9 @@ const config = {
       ),
     },
     extensions: [".js", ".jsx", ".json"],
+    fallback: { path: path.resolve(__dirname, "node_modules/path-browserify") },
```
Comment (stefanv) [inline webpack.config.js:110]: This line looks fine.

File: webpack.config.js (hunk 1 above)
Comment (acrellin) [reply webpack.config.js:89]: This is the only way I could expose the installed `process`; the below fallback approach didn't work here. This appeared to be the most recommended approach for `process` under v5

Comment (stefanv) [reply webpack.config.js:89]: Do you see a problem during runtime?  Because webpack compiles without it.

Comment (acrellin) [reply webpack.config.js:89]: @stefanv Yes, the page crashes with `process is undefined` without this.

Comment (stefanv) [reply webpack.config.js:89]: I cannot reproduce this on the main page.

Comment (acrellin) [reply webpack.config.js:89]: Meaning this branch runs fine locally without this?

Comment (stefanv) [reply webpack.config.js:89]: Yes, as far as I can tell.

Comment (acrellin) [reply webpack.config.js:89]: @stefanv Unless there's a good reason not to include this fix (fails for me locally without this), I propose we move forward here.

Comment (stefanv) [reply webpack.config.js:89]: Sure; how about adding a comment that this should be removed in the future, and then merge (no need to re-run the CI if it works for you locally).

Comment (stefanv) [reply webpack.config.js:89]: Also for the fallback below.

Comment (acrellin) [reply webpack.config.js:90]: Done, thanks!

**Comment by stefanv** [comment]
Thanks, Ari!

**Pull request merged**